### PR TITLE
Scaffolding: Apps

### PR DESCRIPTION
**Pull request by thilton-oddball**
Scaffolding out apps.

### CODE REVIEW

File: app/views/teams/show.html.erb [hunk 1]
```diff
@@ -9,3 +9,9 @@
 
   <%= button_to 'Destroy this team', @team, method: :delete, form: { data: { turbo_confirm: 'Are you sure?' } }, class: 'inline-flex items-center justify-center px-4 py-2 border border-transparent font-medium rounded-md text-white bg-red-900 hover:bg-red-800 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-red-800 sm:text-sm' %>
 </div>
+
+<br>
+
+<div class="mt-4">
+  <%= render 'apps/team_apps' %>
+</div>
```
Comment (LindseySaari) [inline app/views/teams/show.html.erb:17]: You may want to add the extra newline back here

File: test/models/app_test.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class AppTest < ActiveSupport::TestCase
+  # test "the truth" do
```
Comment (LindseySaari) [inline test/models/app_test.rb:6]: Are there any model or controller tests that are applicable?

File: app/views/apps/_app.json.jbuilder [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+json.extract! app, :id, :name, :team_id, :created_at, :updated_at
```
Comment (LindseySaari) [inline app/views/apps/_app.json.jbuilder:3]: Is jbuilder the rails norm? DId we consider Active Model Serializers? Maybe this was also auto generated?

Comment (thilton-oddball) [reply app/views/apps/_app.json.jbuilder:3]: Yes, this is auto generated when running rails generate scaffold App.

**Review by rileyanderson — APPROVED** [review]
LGTM

**Pull request merged**